handsome-hedgehogs/Bangazon-Site
# Wp search

## Status
Choose: **IN DEVELOPMENT**

## Description
Created search bar/method accepting user string input and filtering list of products

## Todos
-[ ] Tests
-[ ] Documentation

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
```
1. git fetch --all
2. git checkout wp-search
3. run app
4. type in search param in search bar ->Enter
5. verify appropriate product show
```

## Impacted Areas in Application
Layout
ProductsController
SearchView

## Link to Feature Ticket
* [#4](https://github.com/handsome-hedgehogs/Bangazon-Site/issues/4) [#5](https://github.com/handsome-hedgehogs/Bangazon-Site/issues/5)
